### PR TITLE
Fix file staging for print_nuclear_contamination

### PR DESCRIPTION
**Pull request by IdoBar**
## PR checklist

- [x] This comment contains a description of changes (with reason).
- [x] If you've fixed a bug or added code that should be tested, add tests!
 - [ ] If you've added a new tool - add to the software_versions process and a regex to `scrape_software_versions.py`
 - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/eager/tree/master/.github/CONTRIBUTING.md)
 - [ ] If necessary, also make a PR on the nf-core/eager _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [x] Make sure your code lints (`nf-core lint .`).
- [x] Ensure the test suite passes (`nextflow run . -profile test,docker`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ### `Fixed`
 
+- [#666](https://github.com/nf-core/eager/issues/666) - Fixed input file staging for `print_nuclear_contamination`
 - [#631](https://github.com/nf-core/eager/issues/631): - Update minimum Nextflow version to 20.07.1, due to unfortunate bug in Nextflow 20.04.1 causing eager to crash if patch pulled
 
+
 ### `Dependencies`
```
Comment (IdoBar) [inline CHANGELOG.md:16]:
```suggestion
### `Dependencies`
```

File: main.nf [hunk 2]
```diff
@@ -2932,7 +2932,7 @@ process print_nuclear_contamination{
     params.run_nuclear_contamination
 
     input:
-    val 'Contam' from ch_from_nuclear_contamination.map { it[7] }.collect()
+    path 'Contam' from ch_from_nuclear_contamination.map { it[7] }.collect()
```
Comment (TCLamnidis) [inline main.nf:2935]:
```suggestion
    path Contam from ch_from_nuclear_contamination.map { it[7] }.collect()
```

Comment (TCLamnidis) [reply main.nf:2935]: with the quotes, the input files are staged as "Contam1", "Contam2",[...]. This breaks the sample names in the output file. 
The omission of the quotes will keep the original file names, and correctly use the Library_IDs in the output file.

**Review by TCLamnidis — CHANGES_REQUESTED** [review]

File: main.nf (hunk 2 above)
Comment (IdoBar) [reply main.nf:2935]: Thanks!
I was wondering why the files came back named that way...

**Review by TCLamnidis — APPROVED** [review]

**Pull request merged**